cat/CaT-service.yaml

```diff
@@ -1776,9 +1776,15 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OCDS_Standards/CCS-OCDS_Schema.yaml#/components/schemas/OrganizationReference'
+                type: object
```

Comment:
It is usually worth extracting these objects into a dedicated schema type to coerce the OpenAPI generator into creating a dedicated class..

Reply:
Done. Please check